modelon/impact/client/entities/case.py

```diff
@@ -43,6 +52,16 @@ def consistent(self) -> bool:
         false otherwise."""
         return self._consistent
 
+    @property
+    def datetime_started(self):
```

Comment:
The type of the returned object is `datetime`, just naming this method `started` would be nicer.